playground/src/screens/Formats.tsx

```diff
@@ -18,6 +21,16 @@ const FormatList = SectionList as SectionListStatic<FormatItem>
 export default function Formats() {
   const client = useRemoteMediaClient()
 
+  useEffect(() => {
+    CastContext.getCastState().then((state) => {
+      console.log('entering formats screen', state)
+    })
+    return () => {
+      console.log('leaving formats screen')
+      CastContext.endCurrentSession(true)
+    }
+  }, [])
```

Comment:
It's fine for now but if you want to set it up as I'm thinking it would be in the future, I'd probably add a new screen called "Session" with different buttons that imperatively manage the current session.